geometryFlags.py

```diff
@@ -0,0 +1,185 @@
+        optimal737 = True;
+        conventional = True
+    if aircraft == 'optimalD8':
+        optimalD8 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+        eng = 3;
+    if aircraft == 'optimal777':
+        optimal777 = True;
+        conventional = True;
+        eng = 4;
+    if aircraft == 'optimal777_M08':
+        optimal777 = True
+        optimal777_M08 = True;
+        conventional = True
+    if aircraft == 'optimal777_M072':
+        optimal777 = True
+        optimal777_M072 = True;
+        conventional = True
+    if aircraft == 'M08D8':
+        M08D8 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+        eng = 3;
+    if aircraft == 'M08D8_noBLI':
+        M08D8_noBLI = True;
+        rearengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'M08_D8_eng_wing':
+        M08_D8_eng_wing = True;
+        wingengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'M072_737':
+        M072_737 = True;
+        conventional = True
+    if aircraft == 'D8_no_BLI':
+        D8_no_BLI = True;
+        rearengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'D8big_M072':
+        D8big = True
+        D8big_M072 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'D8big_M08':
+        D8big = True
+        D8big_M08 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'optimalRJ':
+        optimalRJ = True;
+        conventional = True
+    if aircraft == 'smallD8':
+        smallD8 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+        eng = 3;
+    if aircraft == 'smallD8_eng_wing':
+        smallD8_eng_wing = True;
+        wingengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'smallD8_no_BLI':
+        smallD8_no_BLI = True;
+        rearengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'smallD8_M08_no_BLI':
+        smallD8_no_BLI = True
+        smallD8_M08_no_BLI = True;
+        rearengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'smallD8_M08':
+        smallD8 = True
+        smallD8_M08 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'smallD8_M08_eng_wing':
+        smallD8_eng_wing = True
+        smallD8_M08_eng_wing = True;
+        wingengine = True;
+        piHT = True;
+        doublebubble = True;
+    if aircraft == 'D12':
+        D12 = True;
+        rearengine = True;
+        BLI = True;
+        piHT = True;
+        doublebubble = True;
+        eng = 4;
```

Comment:
great idea here

Reply:
Haven't fully implemented. Will do so in another PR.